tests/unit/meta/test_command.py

```diff
@@ -163,6 +238,19 @@ def test_command_relative_command_found_in_slash(self):
         self.expectThat(cmd.command, Equals("command-foo.wrapper"))
         self.assertThat(wrapper_path, FileExists())
         self.assertThat(wrapper_path, FileContains('#!/bin/sh\nexec /bin/sh "$@"\n'))
+        self.assertThat(
+            self.fake_logger.output.strip(),
+            Equals(
+                "The command 'sh' was not found in the prime directory, it "
+                "has been changed to '/bin/sh'."
+                "\n"
+                "A shell wrapper will be generated for command '/bin/sh' "
```

Comment:
...but you just changed it to '/bin/sh' for me! :)  my command "sh" pattern was relative and did not have those problems!! :thinking: :laughing:

Reply:
It would have only been a non-working snap :shrug: